Vegetation Bending Tutorial (#1757)
o3de/o3de.org: content/docs/learning-guide/tutorials/rendering/_index.md

---
linkTitle: Rendering
title: Rendering in Open 3D Engine

> description: Open 3D Engine performs rendering using the Atom gem.

Reviewer: ```suggestion
description: A collection of rendering tutorials using Atom Renderer in Open 3D Engine (O3DE).
```

---
linkTitle: Rendering
title: Rendering in Open 3D Engine
description: Open 3D Engine performs rendering using the Atom gem.

> menu_uuid: Rendering

Reviewer: ```suggestion
```

toc: true
---

> This section contains tutorials on making custom additions and changes to the Atom renderer. 

Reviewer: ```suggestion
This section contains tutorials on making custom additions and changes to the **Atom Renderer** in **Open 3D Engine (O3DE)**.
```

> Atom Gem is enabled in **Open 3D Engine (O3DE)** by default; for more information, refer to [Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/).

Reviewer: ```suggestion
**Atom Gem** is enabled in O3DE by default. For more information, refer to [Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/).
```
